packages/components/src/__actions__/Button/v3/Button.tsx

```diff
@@ -26,6 +26,12 @@ type ButtonBaseProps = Omit<RACButtonProps, "children"> & {
   iconPosition?: "start" | "end"
   /** Controls if the button inherits width from its parent. @default "false" */
   isFullWidth?: boolean
+  /**
+   * Use the [ReverseColors](https://cultureamp.design/?path=/docs/actions-button-button-v3-api-specification--docs#variants) Provider instead.
+   * @default false
+   * @deprecated Controls the reversed style of Button. This is here to support gradual migration to the ReverseColors Provider.
```

Comment:
Ideally I think we should swap the prop description with the deprecation message.